### ISSUE TITLE
FLAC is played at twice the normal speed 

### ISSUE THREAD

**Issue by moocan** [DeaDBeeF-Player/deadbeef#2039]
### Steps to reproduce the problem
I have some DTS audio tracks with following specs:
```
Input #0, dts, from '03 Personal Jesus.dts':
  Duration: 00:04:50.47, start: 0.000000, bitrate: 1535 kb/s
    Stream #0:0: Audio: dts (DTS 96/24), 96000 Hz, 5.1(side), fltp, 1536 kb/s
```
This audio track is playing well using the DTS format in Deadbeef.
I have converted from DTS format to FLAC format using ffmpeg 4.0.
The FLAC audio track has the following specs:
```
Input #0, flac, from '03 Personal Jesus.flac':
  Metadata:
    ENCODER         : Lavf58.12.100
  Duration: 00:04:55.52, start: 0.000000, bitrate: 7897 kb/s
    Stream #0:0: Audio: flac, 96000 Hz, 5.1(side), s32 (24 bit)
```

### What's going on? Describe the problem in as much detail as possible.

Audio rendering of the FLAC track in Deadbeef seems the be played at twice (or more) the normal speed but but counting down of the playing time in the status bar seems to be normal.
Total length of the track is 00:04:55.52 and Deadbeef is ending the playing of the FLAC audio track at 00:04:39

I have tried to play the same FLAC audio track in VLC or Audacious and audio rendering is normal.

### Information about the software:

Deadbeef version: 0.7.2 installed from [Starws PPA](https://launchpad.net/~starws-box/+archive/ubuntu/deadbeef-player)
OS: Ubuntu Mate 17.10 & 18.04

Kind Regards

**Comment by Lithopsian** [comment]
Sounds like a resampling issue.  Check that you have a resampler in your DSP chain, or that your output can resample automatically.

**Comment by moocan** [comment]
Hello,

Thanks for your answer.
I have the `Resampler (Secret Rabbit Code)` in the DSP chain.
I have tested with and without it.
I tried each `Quality / Algorithm` settings from `SYNC_BEST_QUALITY` to `LINEAR`.
I tried to change between `ALSA output plugin` and `PulseAudio output plugin`.

But always the same result ... no change.
I don't have any additional plugin.

Kr,

**Comment by Alexey-Yakovenko** [comment]
There's not much that we could do about your issue without you providing a test FLAC file, or a full step-by-step instruction how to create one from a WAV file.

**Comment by Alexey-Yakovenko** [comment]
@moocan a screenshot from deadbeef's track properties for that problematic file would be useful too.

**Comment by Alexey-Yakovenko** [comment]
It may also be worth it to try the development build of deadbeef:
https://sourceforge.net/projects/deadbeef/files/travis/linux/

it's very likely that the problem has been fixed.

**Comment by moocan** [comment]
Hello,

Thanks for your quick answer.
Yes I confirm that DTS files works well.
Ok I will send you a mail this evening to the mail address from your account at users.sourceforge.net.
Flac file is heavy, size is about 300MiB. I guess that by mail it will be very difficult. 

I will try your the development build of deadbeef.
Is there any binary or portable edition already built ?

Kr,

**Comment by Alexey-Yakovenko** [comment]
Yes, this is the portable build: https://sourceforge.net/projects/deadbeef/files/travis/linux/master/deadbeef-static_devel-1_x86_64.tar.bz2/download

**Comment by moocan** [comment]
Hello,

I tried the devel build and that's exactly the same thing :(
I tried to play with some standard options like these:
![screenshot](https://user-images.githubusercontent.com/21237706/44026071-35933d6a-9ef2-11e8-83f9-7d4709407d00.png)
which are not available in the 0.7.2-2 build of DeaDBeef deb package.

Kr,
